kube-runtime/src/utils/mod.rs

```diff
@@ -3,11 +3,14 @@
 mod backoff_reset_timer;
 mod event_flatten;
 mod stream_backoff;
+mod stream_subscribe;
```

Comment:
```suggestion
#[cfg(feature = "unstable_runtime_subscribe")]
mod stream_subscribe;
```

should fix the last clippy issues